plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/form/test/index.js

```diff
@@ -182,7 +177,7 @@ describe( 'Form Component', () => {
 		expect( screen.getByLabelText( /Postal code/ ) ).toBeInTheDocument();
 	} );
 
-	it( 'input values are reset after changing the country', async () => {
+	test( 'input values are reset after changing the country', async () => {
```

Comment:
nitpick, but was this change from `it` to `test` actually needed?

Reply:
I understood that `it` and `test` are synonyms. Given that our other tests are using `test`, I applied that change for consistency-reasons.